frontend/src/store/playbackManager.ts

```diff
@@ -270,13 +270,13 @@ class PlaybackManagerStore {
           label: sub.el.DisplayTitle || 'Undefined',
           src:
             sub.el.DeliveryMethod === SubtitleDeliveryMethod.External
-              ? sub.el.DeliveryUrl
+              ? sub.el.DeliveryUrl || undefined
```

Comment:
Adding the `||` undefined isn't great, but I definitely think it's better than casting the entire object below.

Reply:
Why you think it's better?

Reply:
This specifically only defaults the value for the src property on `PlaybackTrack`. If you mean we should change the function to return `PlaybackTrack[]` instead of `PlaybackTrack[] | undefined` by always return an empty list as default, I'm happy to make that change too.